### DISCUSSION TITLE
QRControl is not using the preset name

### DISCUSSION THREAD

**Discussion by bergie5737** [gopro/labs#1195] [Q&A]
Hi. Very new to GoPro. On recommendation, I added GopPro Labs. I've got the QRcontrol on my Android and it reads the QR code, but the custom preset name is not saved. When I use the WebGIU, I get a new preset name, but with QRControl, it seems to change the pre-installed modes on the camera. The QR command does not show the new name in the command, so I guess there is a tick box like on the gui ("Custom named presets") that I miss, but I can't find it. I have a Hero13 Black.

**Comment by dnewman-gpsw** [comment]
There is a bug in QRControl for creating presets. You can do it with web page https://gopro.github.io/labs/control/custom/

**Comment by bergie5737** [comment]
This may be wrong, but works for now. In the QR Command line, I add this line:  oMPRES="0,PresetName" with the number before "PresetName" as the icon selection to display.

**Reply by dnewman-gpsw** [reply]
Yes that works, the app is missing the leading 'o' character.  That will be fixed shortly

**Comment by bergie5737** [comment]
Thanks, that will sort me for now while I figure out how to set up for good footage. One day after I got, it I made an awful timelapse riding my bike. My bog standard phone was better...  I ended up throwing the Gopro of a cliff into water, but that footage is unusable as well.
